### PR TITLE
Allow the configuration of meta and data directories

### PR DESCRIPTION
**Pull request by gthvn1**

### CODE REVIEW

**Review by elemoine — APPROVED** [review]
LGTM. I just think we should document all the configuration params in the README, possibly also with commented lines in the metadata classes. But that's a general comment not specific to this PR.

**Comment by gthvn1** [comment]
Right. I have added a note in the README to enforce the fact that if you specified directories they must exist. I'm not sure if it is the right way. The use case here is if someone has created a 'tmpfs' FS and want to use it for the WAL.

File: README.rst [hunk 1]
```diff
@@ -40,6 +40,21 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb where you specify path for data and metastore. You need
+to ensure that the directories specified for data, wal and metastore exist and
+are set correctly.
```
Comment (elemoine) [inline README.rst]: "are set correctly" is not explicit enough IMO. I'd rather remove it than leaving it as-is.

Comment (simonpasquier) [reply README.rst]: agreed. Also "Single-node influxdb where you specify paths for the data and metastore directories".

**Review by simonpasquier — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (gthvn1) [reply README.rst]: Fixed.

**Review by simonpasquier — APPROVED** [review]

**Pull request merged**